src/ios/GimbalPlugin.m

```diff
@@ -35,7 +35,7 @@
 @implementation GimbalPlugin
 
 // Config keys
-NSString *const GimbalAPIKey = @"com.urbanairship.gimbal_api_key";
+NSString *const GimbalAPIKey = @"com.urbanairship.ios_gimbal_api_key";
```

Comment:
Can we look for both and prefer the platform one? That way we dont break existing usage

Reply:
oh right, I forgot to handle the old way